diffkemp/simpll/ModuleAnalysis.cpp

```diff
@@ -189,17 +189,22 @@ void simplifyModulesDiff(Config &config, OverallResult &Result) {
             if (funPairResult.second.kind == Result::Kind::NOT_EQUAL) {
                 allEqual = false;
                 DEBUG_WITH_TYPE(DEBUG_SIMPLL,
-                                dbgs() << funPairResult.first.first->getName()
-                                       << " are syntactically different\n");
+                                dbgs() << "  " << Color::makeRed("==========")
+                                       << " Found difference between \""
+                                       << funPairResult.first.first->getName()
+                                       << "\" and \""
+                                       << funPairResult.first.second->getName()
+                                       << "\" "
+                                       << Color::makeRed("==========\n"););
```

Comment:
I'd prefer leaving this part as it was and moving this new easy-to-spot notification to the point where function non-equality is detected (i.e. where `} not equal` is printed).

Reply:
Like this?
![image](https://user-images.githubusercontent.com/35199722/134051417-388d7344-466c-4094-aad8-ed3d72ed9b21.png)

Reply:
Yes, great! Feel free to shift the line all the way to the left (i.e. no indent).